UndertaleModTool/Scripts/Builtin Scripts/DebugMsg.csx

```diff
@@ -1,23 +1,23 @@
-﻿ScriptError("This script is temporarily disabled as it doesn't work properly, causing the game to crash, sorry about that.");
-return;
+﻿EnsureDataLoaded();
+ScriptMessage("DebugMsg - Displays dialogue messages\nwhile debug mode is enabled.\n\nAuthor: krzys-h, Kneesnap");
 
-if (false)
-{
-    EnsureDataLoaded();
-    ScriptMessage("DebugMsg - Displays dialogue messages\nwhile debug mode is enabled.\n\nAuthor: krzys-h, Kneesnap");
+var code = Data.GameObjects.ByName("obj_time").EventHandlerFor(EventType.Draw, EventSubtypeDraw.DrawGUI, Data.Strings, Data.Code, Data.CodeLocals);
 
-    var code = Data.GameObjects.ByName("obj_time").EventHandlerFor(EventType.Draw, EventSubtypeDraw.DrawGUI, Data.Strings, Data.Code, Data.CodeLocals);
-
-    code.ReplaceGML(@"
+code.AppendGML(@"
 if (global.debug == 1) {
     draw_set_color(0xFFFF);
```

Comment:
1) I dont think UTMT/GM1 likes the 0x prefix
2) is this a custom color? can it not be replaced with [one of the human readable constants](https://manual.gamemaker.io/monthly/en/GameMaker_Language/GML_Reference/Drawing/Colour_And_Alpha/Colour_And_Alpha.htm)?

Reply:
It was a white color, i replaced this, thx